test/array_serializer_test.rb

```diff
@@ -39,15 +45,52 @@ def test_meta_and_meta_key_attr_readers
         assert_equal @serializer.meta_key, "the meta key"
       end
 
-      def test_json_key_when_resource_is_empty
```

Comment:
Nops, it was introduced on #1007 

> The json_key will become 'postcomments' rather than 'post_comments'.
Using 'underscore' instead of 'downcase' fixes the error.